node/resolvers/catalog/utils.ts

```diff
@@ -114,10 +115,18 @@ export const searchContextGetCategory = async (
   if (!department && !category && !subcategory) {
     return null
   }
-  const url = [department, category, subcategory].filter(Boolean).join('/')
-  const pageType = await catalog.pageType(url)
-
-  if (!typesPossible.includes(pageType.pageType)) {
+  const url = [department, category, subcategory]
+    .filter(Boolean)
+    .map(str => catalogSlugify(str!))
+    .join('/')
+  const pageType = await catalog.pageType(url).catch(() => null)
+  if (!pageType) {
+    logger.warn(
+      `category ${url}, args ${JSON.stringify(args)}`,
```

Comment:
is `args` always stringifyable? (i.e. it never has circular references?)

Reply:
never, nice point tho